### DISCUSSION TITLE
Small images consume too much whitespace

### DISCUSSION THREAD

**Discussion by ChuckJonas** [outline/outline#3013] [General]
I'm running into some less than ideal formatting issues when trying to migrate some docs over:

**Typical Markdown Render**

<img width="620" alt="https___callaway-docs_herokuapp_com_employee-handbook_communication-policy_md" src="https://user-images.githubusercontent.com/5217568/151300655-af8d3342-93a2-494c-bdaf-6975cd92b480.png">

**Outline**
<img width="895" alt="Communication_Policy_-_Callaway_Cloud_Consulting" src="https://user-images.githubusercontent.com/5217568/151300586-d99886f9-8872-42e4-b7fe-70986fae6e36.png">

**Seems there are a couple issue at play:**

1. The alt/caption takes up space even when it is not in use
2. There is an extra line added below the image.  Trying to delete this line only results in the image being deleted.
3. If you manage to get a space above the image in the table, it cannot be removed and the image cannot be moved up (explains why the gmail icon is lower than the other 2)

**Comment by tommoor** [comment]
The space below an image is also filed here – https://github.com/outline/outline/issues/2854 – it's kind of gnarly to fix due to the fact that images are inline masquerading as block level elements, but we'll figure out a way.
